app/[locale]/login/login.tsx

```diff
@@ -15,9 +15,9 @@ const Login = async ({ locale }: { locale: string }) => {
       // bg-logo and bg-background are the same so it removes one of them.
       className={clsx(
         'flex h-auto w-full grow flex-col items-center justify-center',
-        'max-w-[384px] sm:max-w-[512px] md:max-w-[640px] lg:max-w-[768px]',
+        'max-w-[384px] sm:max-w-[512px] md:max-w-[640px] lg:max-w-[640px]',
         'bg-background bg-logo bg-contain bg-center bg-no-repeat bg-blend-overlay',
-        'p-4 sm:p-12 md:p-24'
+        'md:p-18 p-4 sm:p-12 '
```

Comment:
Please run `npm run format`

Reply:
![image](https://github.com/nitkkr-dev/nitkkr/assets/125768789/ffacc62e-a112-46bb-bcf2-2a3c1b0b3e78)
also npm run format doesnt change this

Reply:
pushed now or previously? Because previous was wrong. And whatever your VSCode extension is showing is not following the repo's prettier config.

Reply:
pushed previously, abhi ke lie i bypassed husky and changed this. have a feeling clsx ki wajah se hora tha. idk

Reply:
Just checked this @geekysilento, it happened because `p-18` is an invalid Tailwind class.